app/assets/javascripts/dsds.js.erb

```diff
@@ -0,0 +1,53 @@
+$(document).ready(function () {
+
+    // Clear solicitor search results on ESC keypress
+    $(document).on('keydown', function (event) {
+        if ($('.solicitor_results_list *').length > 0 && event.keyCode === 27) {
+            $('.solicitor_results_list').empty();
+        }
+    });
+
+    var ownSolicitor_checked = ($("input[type='radio']#defence_request_solicitor_type_own:checked").length > 0);
+
+    var own_setup = function () {
+        $('#defence_request_solicitor_firm').removeAttr("disabled");
+        $('#defence_request_solicitor_name').removeAttr("disabled");
+        $('#defence_request_scheme').val("No Scheme");
+        $('#defence_request_scheme').attr("disabled", "disabled");
+    }
+
+    var duty_setup = function () {
+        $('#defence_request_solicitor_firm').attr("disabled", "disabled");
```

Comment:
You could cache these repeated jQuery lookups by saving it to a local variable but it's probably not worth it for alpha